src/platforms/apple/common/performance/instrumentation/automatic-instrumentation.mdx

```diff
@@ -75,6 +69,40 @@ The SDK uses the process start time as the beginning of the app start and the [`
 
 Cold and warm start are Mobile Vitals, which you can learn about in the [full documentation](/product/performance/mobile-vitals).
 
+### Prewarmed App Start Tracking
+
+Starting with iOS 15, the system might [prewarm](https://developer.apple.com/documentation/uikit/app_and_environment/responding_to_the_launch_of_your_app/about_the_app_launch_sequence#3894431) your app by creating the process before the user opens it. In such cases, we won’t be able to reliably measure the app start. But with [sentry-cocoa 7.31.0](https://github.com/getsentry/sentry-cocoa/releases/tag/7.31.0), we’ve introduced a `enablePreWarmedAppStartTracking` feature, (still in its experimental phase), which allows us to collect prewarmed app starts.
+
+Once enabled, the SDK will drop the first app start span if prewarming pauses. This approach will shorten the app start duration, but it will accurately represent the span of time from when a user clicks the app icon to when the app is responsive.
+With this feature, the SDK differentiates between four different app start types:
+
+* __Non-prewarmed cold start__ (See _cold start_  definition above.)
+* __Non-prewarmed warm start__ (See _warm start_  definition above.)
+* __Prewarmed cold start__
+* __Prewrmed warm start__
+
+You can filter for different app start types in Discover with `app_start_type:cold.prewarmed`, `app_start_type:warm.prewarmed`, `app_start_type:cold`, and `app_start_type:warm`.
```

Comment:
Let's link the name of the feature in case someone isn't familiar with it yet
```suggestion
You can filter for different app start types in [Discover](/product/discover-queries/) with `app_start_type:cold.prewarmed`, `app_start_type:warm.prewarmed`, `app_start_type:cold`, and `app_start_type:warm`.
```